app/views/application/_logo_legal_notice.html

```diff
@@ -0,0 +1,3 @@
+Book graphic by <a href="http://www.freepik.com/">Freepik</a> from <a href="http://www.flaticon.com/">Flaticon</a>
```

Comment:
@Alexneisc Because in haml it looks shitty, like:
```haml
Book graphic by
%a(href:"http://...") Freepik
from
%a(href:"http://..." Flaticon
```
And there will be no dynamic content ever. I love haml, but I don't see any sense to use haml here. :cry: